packages/account/src/Sections/Security/DeactivateAccount/deactivate-account-reason.jsx

```diff
@@ -126,25 +126,26 @@ class DeactivateAccountReason extends React.Component {
     validateFields = values => {
         const error = {};
         const selected_reason_count = selectedReasons(values).length;
+
         if (!selected_reason_count) {
             error.empty_reason = localize('Please select at least one reason');
         }
         if ((values.otherTradingPlatforms + values.doToImprove).length > 0 || selected_reason_count) {
             const max_characters = character_limit_no;
             const final_value = preparingReason(values);
 
-            const remaining_characters = max_characters - final_value.length;
-            if (remaining_characters >= 0) {
-                this.setState({ remaining_characters });
-            } else {
-                this.setState({ remaining_characters: 0 });
-            }
+            const text_inputs_length = values.otherTradingPlatforms.length + values.doToImprove.length;
+            let remaining_characters = max_characters - text_inputs_length;
+
+            remaining_characters = remaining_characters >= 0 ? remaining_characters : 0;
 
-            if (!/^[0-9A-z .,'-]*$/.test(final_value)) {
+            this.setState({ remaining_characters });
+
+            if (!/^[0-9A-z .,'-’]*$/.test(final_value)) {
                 error.characters_limits = localize("Must be numbers, letters, and special characters . , ' -");
             }
 
-            if (final_value.length > max_characters) {
+            if (remaining_characters.length > max_characters) {
                 error.characters_count_exceed = 'Please enter no more than 255 characters for both fields.';
```

Comment:
We should change the text `255 characters` if we're reducing it to 110 because the counter below the field will show 110.

Reply:
Good catch I will update the error message as well